collector/memory.go

```diff
@@ -257,247 +256,246 @@ func NewMemoryCollector() (Collector, error) {
 // Collect sends the metric values for each metric
 // to the provided prometheus Metric channel.
 func (c *MemoryCollector) Collect(ctx *ScrapeContext, ch chan<- prometheus.Metric) error {
-	if desc, err := c.collect(ch); err != nil {
+	if desc, err := c.collect(ctx, ch); err != nil {
 		log.Error("failed collecting memory metrics:", desc, err)
 		return err
 	}
 	return nil
 }
 
 type Win32_PerfRawData_PerfOS_Memory struct {
```

Comment:
As with the logical disk collector, would be nice to rename this type

Reply:
Type has been renamed